recipes/backoff/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "backoff" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "330e2cee2525ab49c8bf7d6ec7132635479ecb164c9cc69064c8b37715ff0fad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - backoff
+
+about:
+  home: https://github.com/litl/backoff
+  license: MIT
+  license_family: MIT
+  summary: 'Function decoration for backoff and retry'
+  description: |
+    This module provides function decorators which can be used to wrap a function such that it will be retried until
+    some condition is met. It is meant to be of use when accessing unreliable resources with the potential for
+    intermittent failures i.e. network resources and external APIs. Somewhat more generally, it may also be of use for
+    dynamically polling resources for externally generated content.
+  dev_url: https://github.com/litl/backoff
+
+extra:
+  recipe-maintainers:
+    - gatesn
+    - bgreen-litl
```

Comment:
@bgreen-litl are you ok with being added as a co-maintainer of the `backoff` package on conda-forge?